trino-connector/src/main/java/org/apache/gravitino/trino/connector/GravitinoConfig.java

```diff
@@ -194,7 +194,10 @@ public TrinoConfig() {
           return;
         }
 
-        properties.load(new FileInputStream(configFileName));
+        try (FileInputStream input = new FileInputStream(configFileName)) {
+          properties.load(input);
+        }
+
```

Comment:
Thanks for helping out here. Is this code complete as it looks like the input stream is still not closed?

Reply:
@justinmclean 
The `try/catch` code block will close the `input` automatically, please refer to `try-with-resource`

Reply:
OK, that looks good to me